otter/convergence/model.py

```diff
@@ -563,11 +579,22 @@ def equivalent_definition(self, other_description):
                 other_description.port == self.port)
 
 
+@attr.s
+class CLB(object):
+    """
+    Cloud load balancer
+    """
+    # Is health monitor enabled?
+    health_monitor = attr.ib(default=False)
+
+
 @implementer(ILBNode, IDrainable)
 @attributes([Attribute("node_id", instance_of=basestring),
              Attribute("description", instance_of=CLBDescription),
              Attribute("address", instance_of=basestring),
              Attribute("drained_at", default_value=0.0, instance_of=float),
+             Attribute("status", instance_of=NamedConstant,
```

Comment:
Question: are there any other possible future values here for node status?  My inclination would be to just call this `is_online` and make it a boolean if "online" and "offline" are really the only two; having a whole enumerated `NamedConstant` seems a bit precious if it's really just a boolean.